Core/include/Acts/Seeding/HoughTransformUtils.hpp

```diff
@@ -0,0 +1,380 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// @file HoughTransformSeeder.hpp
+// @author Max Goblirsch, based on work by Riley Xu and Jahred Adelman
+// @brief Implements a set of tools to
+/// implement a hough transform.
+
+#include "Acts/Utilities/Delegate.hpp"
+#include "Acts/Utilities/Logger.hpp"
+#include "Acts/Utilities/Result.hpp"
+
+#include <array>
+#include <map>
+#include <optional>
+#include <set>
+#include <unordered_set>
+
+#include "HoughVectors.hpp"
+
+#pragma once
```

Comment:
I believe this needs to go all the way to the top.

Reply:
thanks, fixed!